features/manage-position/Create.tsx

```diff
@@ -73,6 +78,19 @@ const Create = () => {
   } = empState;
   const liquidationPriceWarningThreshold = 0.1;
 
+  useEffect(() => {
```

Comment:
Can't call `useEffect()` or other hooks inside of conditionals: https://medium.com/@jonchurch/how-to-fix-react-error-rendered-fewer-hooks-than-expected-e6a378985d3c 

Otherwise you can get a `Rendered fewer hooks than expected` error